tools.deps/test-resources/flowstorm/deps.edn

```diff
@@ -1,3 +1,6 @@
-{:aliases {:flowstorm {:classpath-overrides {org.clojure/clojure nil} ;; for disabling the official compiler
-                       :deps                {com.github.jpmonettas/clojure        {:mvn/version "1.12.0-alpha4_3"}
-                                             com.github.jpmonettas/flow-storm-dbg {:mvn/version "3.7.3"}}}}}
+;; Adding a dependency that contains clojure 1.11.1 as a transitive
+{:deps {io.github.nubank/morse {:git/tag "v2023.10.06.02" :git/sha "88b5ff7"}}
+ :aliases
+ ;; let's swap the Clojure compiler by ClojureStorm
+ {:storm {:classpath-overrides {org.clojure/clojure nil} ;; for disabling the official compiler
```

Comment:
I'd prefer an additive change to not lose coverage, e.g. `flowstorm2/deps.edn`

Reply:
Sure, np, but shouldn't that test cover both cases?

Reply:
I prefer to rest assured - to me these are just data - very cheap to accrete

Reply:
Done!